extensions/amp-story/amp-story.md

```diff
@@ -110,6 +110,66 @@ The new attributes:
 | `poster-square-src` | The story poster in square format (1x1 aspect ratio). |
 | `poster-landscape-src` | The story poster in landscape format (4x3 aspect ratio). |
 
+#### `publisher-logo-src` guidelines
+
+The following guidelines apply to the image for the publisher logo:
```

Comment:
Are these guidelines or requirements?  If they are guidelines, then don't use "must", use "should".  If they are requirements, keep the "must".

Reply:
Done